test/test_mixeddofhandler.jl

```diff
@@ -643,6 +643,24 @@ function test_vtk_export()
     rm(filename*".vtu") # clean up
 end
 
+function test_celliterator_on_true_subdomain_smoketest()
+    grid = generate_grid(Hexahedron, (3,3,3))
+
+    dh = DofHandler(grid)
+    sdh = SubDofHandler(dh, [1,2,3])
+    add!(sdh, :u, Lagrange{RefHexahedron,1}()^3)
+    close!(dh)
+
+    for cell in CellIterator(sdh)
+    end
+
+    for cell in CellIterator(dh, [1,2,3])
+    end
```

Comment:
These didn't fail before either, right, so perhaps not required? 
Would also be nice to add a link to the issue - I also find that helpful in understanding why something is smoke-tested.

Reply:
This failed on last release, so you can see this as a smoke test to see if things do at least not crash.

Will add the issue later.